server/simpleFlaskServer1.py

```diff
@@ -0,0 +1,114 @@
+from flask import Flask, url_for, jsonify, request
+import random, requests, sys, json
+app = Flask(__name__)
+
+# should get this from the database
+currentGameId = 50
+numOfQuestionsInAGame = 2
+numOfOptionsPerQuestion = 3
+questionSeeds = {
+        'cities': [ 'mumbai', 'paris', 'new york city', 'rome', ' london' ],
+        'fruits': [ 'apple', 'pear', 'grape', 'orange', 'banana', 'strawberry' ]
+    }
+
+games = {}
+
+imageSearchURL='https://www.googleapis.com/customsearch/v1?key=AIzaSyCeiMGAnozDLIOKhSYCG5lIHbWDjRT6cVg&cx=007408032665432303252:eybewyws4ca'
```

Comment:
you probably don't want to put the key here (since it's public).. 
i ll merge this for now, but you should change your key.